config-default.yml

```diff
@@ -30,11 +30,10 @@ bot:
 style:
     colours:
         blue: 0x3775a8
-        bright_green: 0x01d277
+        green: 0x01d277
         orange: 0xe67e22
         pink: 0xcf84e0
         purple: 0xb734eb
-        soft_green: 0x68c290
```

Comment:
I'm inclined to keep this the same name for the sake of easily referencing it with the [discordpy docs](https://discordpy.readthedocs.io/en/master/api.html#discord.Colour). We do this with all the other colours, so I'm not sure this is a good reason to strafe from that.

This is made worse by the fact that it's actually misleading (rather than just confusing having a non-existent colour) since `discord.Colour.green` is actually 0x2ecc71, not 0x68c290 (which are rather different).